modules/ROOT/pages/xa-transactions.adoc

```diff
@@ -88,6 +88,57 @@ If the `db:insert` operation fails, the transaction is not rolled back. Instead,
 
 Consider that the `jms:publish` inside the `on-error-continue` has another configuration for the connector. This is not required but is possible. With single source (local) Transactions, this is not a valid configuration, since all operations that join a Transaction must use the same Configuration to work as designed.
 
+[[_nested_tx_]]
+== Nesting Transactions
+
+With XA Transaction, you are able to nest transactions. This means, when executing within a transaction, you can create a new Transaction (using the xref:try-scope-concept.adoc[Try Scope]). Therefore, operations that run within the new transaction will be accountable to it. If the nested transaction is rollback, it does not mean that the original one will rollback. After the nested transaction has finished (committed or rolled back), the original transaction continues its execution. 
+
+For example:
+
+[source,xml,linenums]
+----
+<flow name="exampleFlow" >
+	<try transactionalAction="ALWAYS_BEGIN" transactionType="XA">
+		<vm:publish queueName="someVmQueue" config-ref="VM_Config"/>
+		<try transactionalAction="ALWAYS_BEGIN" transactionType="XA"/>
+			<jms:consume config-ref="JMS_Config" destination="someQueue"/>
+			<db:insert config-ref="Database_Config">
+				<db:sql>${insertQuery}</db:sql>
+			</db:insert>
+			<raise-error type="APP:SOME"/>
+		</try>
+	</try>
+	<error-handler>
+		<on-error-continue/>
+	</error-handler>
+</flow>
+----
+
+In the example above, there is an original transaction created by the first `try` Scope. Then, there is a second one created by the `try` Scope within it. So, the `vm:publish` operation runs within the first transaction, the `jms:consume` and the `db:insert` will run within the second transaction. In the example, after the error is raised, the second transaction is rolled back (and so, the two mentioned operations), but since the first try handles the error with an `on-error-propagate`, the first transaction commits (therefore, the `vm:publish` operation). Another example could be:
```

Comment:
"but since the first try handles the error with an `on-error-propagate` " > Is not `on-error-continue` here ?

Reply:
Totally, my bad
```suggestion
In the example above, there is an original transaction created by the first `try` Scope. Then, there is a second one created by the `try` Scope within it. So, the `vm:publish` operation runs within the first transaction, the `jms:consume` and the `db:insert` will run within the second transaction. In the example, after the error is raised, the second transaction is rolled back (and so, the two mentioned operations), but since the first try handles the error with an `on-error-continue`, the first transaction commits (therefore, the `vm:publish` operation). Another example could be:
```